stdlib/src/blackbox_fallbacks/logic_fallbacks.rs

```diff
@@ -5,19 +5,6 @@ use acir::{
     native_types::{Expression, Witness},
 };
 
-// Perform bit decomposition on the provided expression
-#[deprecated(note = "use bit_decomposition function instead")]
```

Comment:
We should double check, this might be a breaking change, since it was marked as pub